### PR TITLE
Fix ie11 node display issues

### PR DESCRIPTION
**Pull request by Gimly**
**What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Nodes were not displayed correctly with IE11/Edge, see issue #12 

**What is the new behavior?**
The graph displays now correctly in all browsers

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**: none

### CODE REVIEW

**Comment by Gimly** [comment]
Any update on this? Is there something blocking the PR? I can make modification if that's the case, thanks.

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ compiled/
 dist/
 build/
 coverage/
+release/
```
Comment (marjan-georgiev) [inline .gitignore]: We need to have the release directory in the repository. Let's remove this line please. However, do not commit anything in the release directory. We generate that when we release a new version.

**Comment by Gimly** [comment]
@marjan-georgiev OK, I did the change in the .gitignore. I understand you need the folder, but you should maybe consider doing something like that:
https://stackoverflow.com/a/932982/123597 to keep the folder but make it clear that everything inside should be ignored. Otherwise, you might have people commiting their test publish.

**Comment by marjan-georgiev** [comment]
We actually need the contents inside that folder. That is what is used when you install the library in your project.

**Comment by Squashy83** [comment]
Hi guys any news on this matter? when I open https://swimlane.github.io/ngx-graph/ on IE11 still showing no links between nodes and when I try to drag a node SVG4601 error comes out..

please help

**Comment by nicosaliagas** [comment]
Same error on IE11

Really need some help please

**Pull request merged**